### PR TITLE
Added testing

### PR DESCRIPTION
**Pull request by SmartManoj**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,27 +40,45 @@ A minimal Quart example is,
 
 .. code-block:: python
 
-    from quart import Quart, websocket
```
Comment (pgjones) [inline README.rst]: Why change the indentation here? Does it display incorrectly somewhere?

File: README.rst [hunk 2]
```diff
@@ -40,27 +40,45 @@ A minimal Quart example is,
...
-    app.run()
+   app.run()
 
 if the above is in a file called ``app.py`` it can be run as,
 
 .. code-block:: console
 
     $ python app.py
 
+Then run the following html file to test
```
Comment (pgjones) [inline README.rst]: I'm not sure this is required in the Quickstart (also as the code doesn't use it), the tutorials should cover this.

Comment (SmartManoj) [reply README.rst]: the code doesn't use it ?

This previous example result is done by flask itself.

**Comment by pgjones** [comment]
Thanks, but I don't think this is required in the README, which I'd like to keep as short as is feasible.

**Pull request closed without merging**